### PR TITLE
feature(sdk): Fixed loading multiple plugins to have different scopes

### PR DESCRIPTION
**Pull request by Daboo1998**

### CODE REVIEW

**Review by PAKovalev — APPROVED** [review]

File: packages/plugin_miro/public/manifest.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "short_name": "React App",
+  "plugin_id": "plugin_miro",
```
Comment (dmitry-yudakov) [inline packages/plugin_miro/public/manifest.json:3]: I actually though of having another `manifest.json` in the root of the plugin repo with plugin related stuff - id, description, icon, etc.

This one is for CRA which is for dev only. We could even remove it I guess - I don't think it's used for web normally.

**Review by dmitry-yudakov — APPROVED** [review]
I generally approve except for the manifest suggestion.

File: packages/plugin_miro/public/manifest.json (hunk 1 above)
Comment (Daboo1998) [reply packages/plugin_miro/public/manifest.json:3]: So you suggest to move it to root and remove other fields than plugin_id?

Comment (dmitry-yudakov) [reply packages/plugin_miro/public/manifest.json:3]: Yep, something like that - with fields `id, name, icon, description`.

**Comment by Daboo1998** [comment]
yeah lets have it consistent, at least for now

**Pull request merged**